Floorplan.py

```diff
@@ -7,8 +8,13 @@ class Floorplan(object):
     The floorplan the DotBots move in.
     '''
     
-    def __init__(self, drawing):
+    def __init__(self, drawing, alias=None):
+
+        if alias:
```

Comment:
`assert not drawing`

You want to make sure you cannot instantiate the class by specifying both a drawing and an alias